core/block_files/blockchain.py

```diff
@@ -132,6 +183,9 @@ def get_current_index(self) -> int:
     @property
     def to_dict(self):
```

Comment:
Is this different than calling the classes `__dict__` method?

Reply:
yes, the __dict__ method no longer works